README.md

```diff
@@ -71,6 +71,9 @@ The official ML.NET samples are divided in multiple categories depending on the
     <td align="middle"><img src="images/spike-detection.png" alt="Spike detection chart"><br><img src="images/app-type-getting-started-term-cursor.png" alt="Getting started icon"><br><b>Power Anomaly Detection<br><a href="samples/csharp/getting-started/AnomalyDetection_PowerMeterReadings">C#</a><b></td>
       <td align="middle"><img src="images/anomaly-detection.png" alt="Power Anomaly detection chart"><br><img src="images/app-type-getting-started-term-cursor.png" alt="Getting started icon"><br><b>Credit Card Fraud Detection<br>(Anomaly Detection)<br><a href="samples/csharp/getting-started/AnomalyDetection_CreditCardFraudDetection">C#</a><b></td>
   </tr>
+  <tr>
+    <td align="middle"><img src="images/spike-detection.png" alt="Spike detection chart"><br><img src="images/app-type-getting-started-term-cursor.png" alt="Getting started icon"><br><b>Anomaly Detection of Numbers of Phone Calls<br><a href="samples/csharp/getting-started/AnomalyDetection_PhoneCalls">C#</a><b></td>
+  </td>
```

Comment:
I'm not sure there's something clean to do about it, though inserting a table row changes the grey/white of title-row/example-row to invert.

https://github.com/dotnet/machinelearning-samples/blob/b2daa2a9d72be47e44b5a296c6d607f39d90d4f4/README.md

We could make each task a new table, and explicitly set the `td` width to keep the table columns aligned.